mmdet/apis/ote/apis/detection/ote_utils/misc.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import importlib
+import os
+import tempfile
+from typing import Optional
+
+import yaml
+from ote_sdk.entities.train_parameters import UpdateProgressCallback
+from ote_sdk.usecases.reporting.time_monitor_callback import TimeMonitorCallback
+
+
+def load_template(path):
+    with open(path) as f:
+        template = yaml.full_load(f)
+    return template
+
+
+def get_task_class(path):
+    module_name, class_name = path.rsplit('.', 1)
+    module = importlib.import_module(module_name)
+    return getattr(module, class_name)
+
+
+def reload_hyper_parameters(model_template):
```

Comment:
rm!

Reply:
Please propose corresponding changes to sample and tests.

Reply:
https://github.com/openvinotoolkit/mmdetection/pull/163/files

Reply:
rebase/merge

Actually my comment is redundant, I just forgot that I've did it already

Reply:
I see, my bad here.